src/views/AmalgamationSelection.vue

```diff
@@ -0,0 +1,265 @@
+export default class AmalgamationSelection extends Vue {
+  @Getter(useBusinessStore) getLegalType!: string
+  @Getter(useBusinessStore) isBComp!: boolean
+  @Getter(useBusinessStore) isBcCompany!: boolean
+  @Getter(useBusinessStore) isCcc!: boolean
+  @Getter(useBusinessStore) isUlc!: boolean
+
+  /** Get the amalmagated company result name depending on the type. */
+  getRegularAmalgamationText (): string {
+    if (this.isBComp || this.isBcCompany) return 'BC limited company'
+    if (this.isCcc) return 'BC community contribution company'
+    if (this.isUlc) return 'BC unlimited liability company'
+  }
+
+  /** Start Regular Long-form button pressed. */
+  async startRegularAmalgamation (): Promise<any> {
+    // let legalType = null
+    // if (this.isBComp) legalType = CorpTypeCd.BC_COMPANY
+    // else legalType = this.getLegalType
+
+    try {
+      // show spinner since this is a network call
+      this.$root.$emit('showSpinner', true)
+      // const accountId = +JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
+      // const businessId = await this.createBusinessAA(accountId, legalType)
+      return
+    } catch (error) {
+      this.$root.$emit('showSpinner', false)
+      throw new Error('Unable to Amalgamate Now ' + error)
+    }
+  }
+
+  /**
+   * Creates (posts) a draft (temporary) business record.
+   * Must be logged in to use this.
+   * Throws an exception on error.
+   */
+  async createBusiness (businessRequest: any): Promise<any> {
+    const axios = addAxiosInterceptors(Axios.create())
+    const legalApiUrl = sessionStorage.getItem('LEGAL_API_URL')
+
+    const url = `${legalApiUrl}/businesses?draft=true`
+    return axios.post(url, businessRequest)
+  }
+
+  /**
+   * Create a draft amalgamation application based on selected business type.
+   * @param accountId Account ID of logged in user.
+   * @param legalType The legal type of the amalgamated business
+   */
+  async createBusinessAA (accountId: number, legalType: CorpTypeCd): Promise<string> {
+    const businessRequest = {
+      filing: {
+        header: {
+          name: FilingTypes.AMALGAMATION_APPLICATION,
+          accountId: accountId
+        },
+        business: {
+          legalType: legalType
+        },
+        amalgamation: {
+          nameRequest: {
+            legalType: legalType
+          },
+          type: AmalgamationTypes.REGULAR
+        }
+      }
+    } as any
+
+    const createBusinessResponse =
+      await this.createBusiness(businessRequest).catch(error => {
+        throw new Error('Unable to create new Amalgamation Draft ' + error)
+      })
+
+    return createBusinessResponse.data?.filing?.business?.identifier as string
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+header p,
+section p,
+ul {
+  color: $gray7;
+}
+
+h1 {
+  margin-bottom: 1.25rem;
+  line-height: 2rem;
+  letter-spacing: -0.01rem;
+}
+
+// Have all the 3 options (cards) with same height.
```

Comment:
Took me some time to perfect this.

Because one card had more text than the other, the sizes and positioning were all messed up.

The code here and below does multiple things. It forces all the cards to be of same size and positions the buttons all at the same level by centering it in the card and pushing it to the bottom.

Tested this also for smaller screens, works well!

Smaller screen:
![smaller screens](https://github.com/bcgov/business-filings-ui/assets/122301442/f0fa0d28-21e2-4b9f-a6d7-65700673ac72)

Phones:
![phones screen](https://github.com/bcgov/business-filings-ui/assets/122301442/24698fa8-60ad-4cb8-bfbd-bbc9c3c48724)